Automattic/jetpack
# Instant Search: Add TrainTracks analytics

#### Changes proposed in this Pull Request:
This adds TrainTracks analytics calls to Instant Search results.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Yes, it adds analytics calls for rendering and interacting with Instant Search results.

#### Testing instructions:
* Add `define( "JETPACK_SEARCH_PROTOTYPE", true );` to your wp-config.php.
* Ensure that your site has the Jetpack Pro plan and Jetpack Search enabled.
* Add a Jetpack Search widget to the Search page sidebar.
* Open your devtools network inspector and select a checkbox labeled `Preserve Log`.
* Enter a query into the site's search widget.
* Filter for `stats.wp` requests in the network inspector and ensure that you see a request for `stats.wp.com/w.js?60`.
* Filter for `t.gif` requests in the network inspector and ensure that you see 10 calls for `t.gif` with `_en` param equaling `jetpack_instant_search_traintracks_render`.
* Click on a search result link.
* Ensure that you see one additional call for `t.gif` with `_en` param equaling `jetpack_instant_search_traintracks_interact`.

#### Proposed changelog entry for your changes:
None.
